libcxx/docs/ReleaseNotes/21.rst

```diff
@@ -43,6 +43,7 @@ Implemented Papers
 - P1361R2: Integration of chrono with text formatting (`Github <https://github.com/llvm/llvm-project/issues/100014>`__)
 - P2255R2: A type trait to detect reference binding to temporary (implemented the type traits only) (`Github <https://github.com/llvm/llvm-project/issues/105180>`__)
 - P2562R1: ``constexpr`` Stable Sorting (`Github <https://github.com/llvm/llvm-project/issues/105360>`__)
+- P0448R4: A ``strstream`` replacement using ``span<charT>`` as buffer (`Github <https://github.com/llvm/llvm-project/issues/105145>`__)
```

Comment:
Is it conventional to append the new entry of the implemented paper to the end of this list, but not insert it in alphabetical order (i.e. between those of N4258 and P0767R1 at this moment)?

Reply:
I think people just add the new entries at the end of the list normally. In the previous releases I'd stuck mine entries next to each other to avoid merge conflicts in the days before branching.